platform/contracts/admin/src/msg/mod.rs

```diff
@@ -53,9 +59,29 @@ pub enum ExecuteMsg {
     EndOfMigration {},
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize, JsonSchema)]
+pub struct MigrateLeases
+where
+    Uint64: Into<CodeId>,
+    CodeId: Into<Uint64>,
```

Comment:
what's the reason of specifying this?

Reply:
Static assertions that the types are indeed interchangeable.